linkup-cli/src/local_server.rs

```diff
@@ -187,39 +175,28 @@ async fn linkup_request_handler(
         Ok(result) => result,
         Err(_) => {
             return HttpResponse::UnprocessableEntity()
-                .append_header(header::ContentType::plaintext())
+                .append_header(ContentType::plaintext())
                 .body("Unprocessable Content - local server")
         }
     };
 
-    let (dest_service_name, destination_url) =
-        match get_target_service(url.clone(), headers.clone(), &config, &session_name) {
-            Some(result) => result,
-            None => {
-                return HttpResponse::NotFound()
-                    .append_header(header::ContentType::plaintext())
-                    .body("Not target url for request - local server")
-            }
-        };
-
-    let mut extra_headers =
-        get_additional_headers(url, &headers, &session_name, &dest_service_name);
+    let target_service = match get_target_service(&url, &headers, &config, &session_name) {
+        Some(result) => result,
+        None => {
+            return HttpResponse::NotFound()
+                .append_header(ContentType::plaintext())
+                .body("Not target url for request - local server")
+        }
+    };
 
-    // TODO(ostenbom): Consider moving host override into additional_headers function
-    extra_headers.insert(
-        "host".to_string(),
-        Url::parse(&destination_url)
-            .unwrap()
-            .host_str()
-            .unwrap()
-            .to_string(),
-    );
```

Comment:
Hmmm, that is a good point. Maybe to avoid changing functionality on this PR I can keep the adding of the headers in the same place and maybe tackle this on a follow up if we want to merge into the `get_additional_headers` 🤔

Reply:
Changed on f4cf4b74054bbc7a54801032127ab4b437372ec6